src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -314,6 +315,11 @@ public void setMaxRecordsInRam(final int maxRecordsInRam) {
         this.maxRecordsInRam = maxRecordsInRam;
     }
 
+    /** Allows the caller to decide whether or not to write the PG tag to reads when applicable */
```

Comment:
For a docstring, I think it's better to describe behavior of the property, e.g. `Set addPGTagToReads. If true, the PG will be added to reads when ... If false, the PG tag will not be added.`